frontend/ui/widget/infomessage.lua

```diff
@@ -25,6 +27,14 @@ local InfoMessage = InputContainer:new{
     face = Font:getFace("infofont", 25),
     text = "",
     timeout = nil, -- in seconds
+    width = nil,  -- The width of the InfoMessage. Keep it nil to use default value.
+    height = nil,  -- The height of the InfoMessage. If this field is set, a scrollbar may be shown.
+    image = nil,  -- The image shows at the left of the InfoMessage.
+    image_width = nil,  -- The image width if image is used. Keep it nil to use original width.
+    image_height = nil,  -- The image height if image is used. Keep it nil to use original height.
+    -- Whether the default image should be shown. If it is false, and image is nil, there will be no
+    -- image.
+    default_image = true,
```

Comment:
Then the meaning of this flag would be changed. I do not have concern, but the following logic will also be updated.